cmd/ftl/cmd_init.go

```diff
@@ -71,7 +71,7 @@ func (i initGoCmd) Run(ctx context.Context, parent *initCmd) error {
 	if err := updateGitIgnore(i.Dir); err != nil {
 		return err
 	}
-	if err := projectconfig.MaybeCreateDefault(ctx); err != nil {
+	if err := projectconfig.MaybeCreateDefault(ctx, i.Dir); err != nil {
```

Comment:
I think we might want to create the project file in the gitRoot folder vs. where this command was run. I just tried this in the FTL repo and it added the project file to `examples/go` since that's where I ran the `ftl init....` command. But I would guess we would want the file to be created at the root of the ftl repo.

Reply:
I agree; think we want the search for `ftl-project.toml` to end at the gitroot instead of walking up the tree to the fs root, then it would create it there if it doesn't exist. Was planning on bringing this up during standup, but I'm also wondering about the general assumption that the project is in a git repo. My test project wasn't and FTL was very angry.

Reply:
@wesbillman added an explicit check for a git repo during `init`

Reply:
Nice. Thanks!

Maybe we should just list the assumptions and make sure the code does what we expect. So maybe the process is something like this in order (please feel free to make changes or add assumptions here)

1. Check for existing ftl-project.toml.
2. Create new ftl-project.toml at the gitRoot if gitRoot is found
3. Create new ftl-project at the location where the `ftl init` command was run

I don't suspect we would ever want to create this at the FS root.

Reply:
Chatted with @safeer and he's going to put together an RFC for changing the CLI to have more obvious commands for creating ftl projects and modules.